fonzie/views/user.py

```diff
@@ -27,12 +29,17 @@ def get(self, request, version, format=None):
         """
         user = request.user
         issued_at = datetime.utcnow()
+        try:
+            language = user.preferences.get(key="pref-lang").value
+        except ObjectDoesNotExist:
+            language = None
```

Comment:
```suggestion
            language = None
```
Could we have the current language here instead?

Reply:
Indeed. I used the `LANGUAGE_CODE` of the app as fallback as this the current behavior of OpenEdX.